install/kubernetes/cilium/charts/preflight/templates/clusterrole.yaml

```diff
@@ -55,11 +55,7 @@ rules:
   resources:
   - customresourcedefinitions
   verbs:
-  - create
   - get
-  - list
-  - watch
-  - update
```

Comment:
This can't be removed, it can only be deprecated. Mark this for removal in 1.10. See 750c7bc3040b858f30af2b1957b30bdd5ad70c56 for more info.